flopy/utils/gridintersect.py

```diff
@@ -233,6 +235,7 @@ def intersect(self, shp, **kwargs):
         shp = gu.shapely
         sort_by_cellid = kwargs.pop("sort_by_cellid", True)
         keepzerolengths = kwargs.pop("keepzerolengths", False)
+        keep_stacked_cells = kwargs.pop("keep_stacked_cells", False)
```

Comment:
Do we want to keep this syntax with **kwargs, or should we add the keyword arguments to the intersect method explicitly? (Even if they're sometimes not used, depending on the GridIntersect settings).

Reply:
I would vote for an actual argument rather than a `**kwargs`. I would leave `**kwargs` that are passed on to lower level functions like `matplotlib`.